convene-web/app/javascript/src/custom_components/header.scss

```diff
@@ -1,7 +1,7 @@
 @layer components {
   .header {
     .space-menu {
-      @apply flex items-center justify-between px-4 py-3 bg-blue-500 text-white;
+      @apply flex items-center justify-between px-4 py-3 bg-purple-900 text-white;
```

Comment:
Not Blocking: We may want to pull the coloring into something more abstract, so that if we do another re-color we don't have to perform shotgun surgery

For example: `bg-primary-900` could be an alias of `bg-purple-900` and if we wanted to switch to cyan, we could change `bg-primary-900` to point to `bg-cyan-900` instead of `bg-purple-900`.

My preference is we identify 3~6 hues and give them formal names that indicate their use; and we choose which color to use in which context. I would probably want @colombene to define these, since she's doing the majority of our UI work.

For example, I could see us using `dominant` or `primary` to describe the hue we use most prominently throughout the design, and `complement` or `accent` to describe the color that gives contrast, and `supplement` to describe a color that we use in cases where we want something that isn't high-contrast but also isn't the same as the dominent hue.

What I'm hoping is that the words in our styling code to be tied relatively closely [to the application of color theory](https://uxplanet.org/color-glossary-for-designers-terms-and-definitions-1a65549804bd), so that it acts as a "signpost" that helps guide folks learning CSS _from_ design (or learning design _from_ css)

Reply:
Yes, we can also aggregate all these color in `tailwind.config.js`.
https://tailwindcss.com/docs/customizing-colors#naming-your-colors
@roseaboveit and I talked about extracting these but we were sure the exact color code so we put purple here for now.
We can bring up the need for identifying a few color and name those as `neutral`, `info`, `alert` or `primary` and have all those color scheme store in this config js file for easy global edit.

There might be a need to implement custom theme for different workspace but I think we can worry about that later.